pkg/vault/ledger/README.md

```diff
@@ -1,5 +1,6 @@
 # Ledger vault
-
+Connect the ledger device to the system in which signatory is running.
+Install tezos-wallet and tezos-baker apps from ledger live. 
```

Comment:
I think this should be expanded to include something about developer settings being activated in order to have access to download the tezos-baker app

Reply:
Added a line for that and also added the link on how to change to developer mode.